impeller/geometry/geometry_unittests.cc

```diff
@@ -640,6 +640,7 @@ TEST(GeometryTest, CanGenerateMipCounts) {
   ASSERT_EQ((Size{128, 0}.MipCount()), 1u);
   ASSERT_EQ((Size{128, -25}.MipCount()), 1u);
   ASSERT_EQ((Size{-128, 25}.MipCount()), 1u);
+  ASSERT_EQ((Size{1, 1}.MipCount()), 1u);
```

Comment:
Mip counts should always at least be 1. Perhaps also check `Size{0, 0}`?

Reply:
Done